### ISSUE TITLE
Problems in target indices?

### ISSUE THREAD

**Issue by dmhyun** [LeonCrashCode/AttentionTargetSentiment#4]
Hi, I found something strange in target indices (from, to index) on datasets.

First line of T_data/train.posneg is:
```
shaquille o'neal to miss 3rd straight playoff game | The ... : shaquille o'neal will miss his third straight play ... . ||| 11 12 ||| -1
```
If we select a target using the indices (starting from 0), we get ': shaquille'. However the right target is 'shaquille o'neal'.

Here are other examples of strange targets in the T_data/test.posneg file:
```
: martha
news windows
. mandy
name harry
- britney
in harry
having windows
```

Also, strange examples in Z_data/tst.dat file:
```
the government's poor response to Suhakam's numerous recommendations thus far
returning home soon to continue terrorizing the American people
The
```
I can not sure above examples of Z_data are right.

Could you confirm that these(T_data, Z_data) are correct?

**Comment by LeonCrashCode** [comment]
Thank you for pointing out these stranges. I have checked the data and found that indeed the T_data has some problems, i.e. some indices are not correct, which is caused by the fact that some of sentences have more than one target in original data. So I formatted the original data incorrectly, and sorry for bringing strangeness. But the Z_data is correct (I got confirmation from the author I followed). Actually, Z_data has contained some sentences from T_data, and it is more reasonable because it has train/dev/test split. I strongly recommend to use Z_data for experiments.
